tests/utimensat/01.t

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+# $FreeBSD$
+
+desc="utimensat with UTIME_NOW will set the will set typestamps to now"
+
+dir=`dirname $0`
+. ${dir}/../misc.sh
+echo "1..7"
+
+require utimensat
```

Comment:
Actually, I'll just delete it, because utimensat is part of POSIX and AFAIK everything supports it (but I don't have a OSX machine to check).  I'll add a separate require statement for the birthtime-specific tests.